src/index.js

```diff
@@ -0,0 +1,34 @@
+const BigNumber = require("bignumber.js");
```

Comment:
This seems to be a clashing dependency with the other repo. I guess this is only used for its `isZero()` in your filter map. Are we even sure if `isZero` will properly capture our notion of "zeroness"?

Reply:
the reason for using this, is that we use that for big numbers with decimals. Otherwise we use `BN`.

Probably here we would be fine parsing as a float. 
But I just feel it was more correct to convert it to a bignumber, and check for Zero.

Reply:
https://github.com/MikeMcl/bignumber.js/